content/en/docs/reference/glossary/priority-class.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Priority Class
+id: priority-class
+date: 2023-07-10
+full_link: docs/concepts/configuration/pod-priority-preemption/#priorityclass
+short_description: >
+ the PriorityClass is a resource that allows you to assign priorities to different pods within a cluster It helps the scheduler prioritize the scheduling of pods based on their importance or criticality.
+aka:
+tags:
+- operation
+---
+ A Priority Class is an object that is not namespaced and it maps a priority class name to a priority value represented as an integer.
+<!--more-->
+The [PriorityClass](docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityClass) is a non-namespaced object that defines a mapping from a priority class name to the integer value of the priority. The name of the priority is specified in the `name` field, and the value of the priority is specified in the `value` field. The higher the value, the higher the priority. The value of PriorityClass object can be in range from -2147483648 to 1000000000 inclusive. 
```

Comment:
The link is not correct needs a minor correction. Refer below suggestion. 

Also, would be nice to wrap the long lines of text and remove the extra-blank lines from end of this file as request in the [earlier review comment](https://github.com/kubernetes/website/pull/41972/files#r1261845097).

```suggestion
The [PriorityClass](/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityClass) is a non-namespaced object that defines a mapping from a priority class name to the integer value of the priority. The name of the priority is specified in the `name` field, and the value of the priority is specified in the `value` field. The higher the value, the higher the priority. The value of PriorityClass object can be in range from -2147483648 to 1000000000 inclusive. 
```

Reply:
I actually had /docs in my earlier PR but I was suggested to change it to docs

Reply:
@EraKin575 Refer https://github.com/kubernetes/website/pull/41972#discussion_r1266549586 to validate if the current suggested link in the PR is working or not.